core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -569,13 +571,17 @@ void postCommit() {
           }
 
           @Override
-          ClientStream newSubstream(ClientStreamTracer.Factory tracerFactory, Metadata newHeaders) {
-            CallOptions newOptions = callOptions.withStreamTracerFactory(tracerFactory);
+          ClientStream newSubstream(
+              Metadata newHeaders, ClientStreamTracer.Factory factory, boolean isTransparentRetry) {
+            CallOptions newOptions = callOptions;
```

Comment:
Remove line? That'd revert the following line back to the way it was.

Reply:
Done.